ios/FluentUI.Demo/FluentUI.Demo/Demos/OtherCellsDemoController.swift

```diff
@@ -76,28 +54,34 @@ extension OtherCellsDemoController: DemoAppearanceDelegate {
     }
 
     // MARK: - Custom tokens
-    private class ThemeWideOverrideActionsCellTokens: ActionsCellTokens {
-        override var mainBrandColor: DynamicColor {
-            // "Charcoal"
-            return DynamicColor(light: GlobalTokens().sharedColors[.charcoal][.tint50],
-                                dark: GlobalTokens().sharedColors[.charcoal][.shade40])
-        }
+    private var themeWideOverrideTableViewCellTokenSet: [TableViewCellTokenSet.Tokens: ControlTokenValue] {
+        return [
+            .mainBrandColor: .dynamicColor {
+                // "Charcoal"
+                return DynamicColor(light: GlobalTokens().sharedColors[.charcoal][.tint50],
+                                    dark: GlobalTokens().sharedColors[.charcoal][.shade40])
```

Comment:
I guess it was in the original code, but do we really need the `// "Charcoal"` comment when the next two lines both have `GlobalTokens().sharedColors[.charcoal]`?

Reply:
⬜

Eh, probably not "needed" but yeah I don't think it's hurting anything enough to warrant removing.